06-TimeManagement/06_timeMan/timeMan.c

```diff
@@ -0,0 +1,151 @@
+#include <linux/init.h>
+#include <linux/module.h>
+#include <linux/device.h>
+#include <linux/err.h>
+#include <linux/time.h>
+#include <linux/timer.h>
+
+#define MODULE_NAME "timeMan"
+
+//extern unsigned long volatile jiffies;
```

Comment:
@Xanchan 
Please clean up all unnesessary comments and dead code.

Reply:
needed fixes done.